test/common/protobuf/utility_test.cc

```diff
@@ -1357,92 +1358,72 @@ TEST_F(ProtobufUtilityTest, ValueUtilLoadFromYamlObjectWithIgnoredEntries) {
       "struct_value { fields { key: \"baz\" value { string_value: \"qux\" } } }"));
 }
 
+std::string toJson(const ProtobufWkt::Value& v) {
+  std::string json_string;
+  ValueUtil::appendJsonToString(v, json_string);
+  return json_string;
+}
+
 TEST_F(ProtobufUtilityTest, ValueToJson) {
   {
     // null
     ProtobufWkt::Value v;
-
-    std::string json_string;
-    ValueUtil::toJsonString(v, json_string);
-    EXPECT_EQ(json_string, "null");
-    json_string.clear();
+    EXPECT_EQ(toJson(v), "null");
 
     v.set_null_value(ProtobufWkt::NULL_VALUE);
-    ValueUtil::toJsonString(v, json_string);
-    EXPECT_EQ(json_string, "null");
+    EXPECT_EQ(toJson(v), "null");
   }
 
   {
     // bool
     ProtobufWkt::Value v;
     v.set_bool_value(true);
-
-    std::string json_string;
-    ValueUtil::toJsonString(v, json_string);
-    EXPECT_EQ(json_string, "true");
-    json_string.clear();
+    EXPECT_EQ(toJson(v), "true");
```

Comment:
this is a lot nicer. Do we need to declare a new ProtobufWkt::Value in each section? Or would this have the same effect:
```
ProtobufWkt::Value v;
EXPECT_EQ(toJson(v), "null");
v.set_bool_value(true);
EXPECT_EQ(toJson(v), "null");
v.set_number_value(1);
EXPECT_EQ(toJson(v), "1");
...
```
alternatively you could put `ProtobufWkt::Value value_;` in the test fixture and have every one of these stanzas be a separate 1 or 2-line TEST_F.

Reply:
SGTM.